examples/tfengine/simple.yaml

```diff
@@ -57,7 +57,9 @@ templates:
     MONITOR:
       PROJECT_ID: "example-monitor"
       DOMAIN: "example.com"
-    ORG_POLICIES: {} # Use default org policies.
+    ORG_POLICIES:
+      POLICY_FOR: organization
+      NODE_ID: "12345678"
```

Comment:
Is RESOURCE_ID clearer?

Reply:
What about ROOT_ID and ROOT_TYPE (or ANCESTRY_ID and ANCESTRY_TYPE). Resource sounds more like a vm, bucket, etc.

Reply:
ROOT sgtm.

Another option could be PARENT, that's what's used in https://github.com/terraform-google-modules/terraform-google-log-export

Reply:
That's a good example. I'll be consistent with that.